tests/performance-test/deploy/performance-test-job-tg.yml.template

```diff
@@ -14,7 +14,7 @@ spec:
       restartPolicy: Never
       containers:
         - name: performance-test
-          image: <<REGISTRY_INFO>>/sa-telemetry/performance-test:dev
+          image: quay.io/plmr/performance-test:dev
```

Comment:
Do you want access to the quay.io/redhat-service-assurance to host the container image in that namespace instead?

Reply:
Actually looks like both Chris and yourself are members there, so should be able to push there. Not saying you have to, but you do have that as an option.

Reply:
Oh yeah I could. Doesn't really matter since this one wont be around long